pkg/controller/pravega/pravega_controller.go

```diff
@@ -179,15 +191,15 @@ func MakeControllerConfigMap(p *api.PravegaCluster) *corev1.ConfigMap {
 		javaOpts = append(javaOpts, fmt.Sprintf("-D%v=%v", name, value))
 	}
 
+	authEnabledStr := fmt.Sprint(p.Spec.Authentication.IsEnabled())
```

Comment:
Is it typical to use `Str` suffix to denote a String variable? It looks odd to me.

Reply:
configMap in kubernetes is a map[string,string]. So this is dictated by kubernetes. 
Earlier value also was a string and was hard coded to "false" which is wrong.